android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -300,6 +361,84 @@ private MediaSource buildMediaSource(Uri uri, String overrideExtension) {
         }
     }
 
+    private DefaultDrmSessionManager createDrmSessionManager(ActionToken drmParam) throws UnsupportedDrmException {
```

Comment:
Could be marked as `@Nullable`

Reply:
I have made it nonnull as it is not possible to create drm manager without drm params.